Roles/AddOns/Common/Sloth.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AmongUs.GameOptions;
+using static TOHE.Options;
+
+namespace TOHE.Roles.AddOns.Common;
+
+public class Sloth : IAddon
+{
+    private const int Id = 29700;
+    public AddonTypes Type => AddonTypes.Harmful;
+
+    private static OptionItem OptionSpeed;
+
+    public void SetupCustomOption()
+    {
+        SetupAdtRoleOptions(Id, CustomRoles.Sloth, canSetNum: true, tab: TabGroup.Addons, teamSpawnOptions: true);
+        OptionSpeed = FloatOptionItem.Create(Id + 10, "SlothSpeed", new(25f, 75f, 5f), 50f, TabGroup.Addons, false)
+            .SetParent(CustomRoleSpawnChances[CustomRoles.Flash])
+            .SetValueFormat(OptionFormat.Multiplier);
+    }
+    public static void SetSpeed(byte playerId, bool clearAddOn)
+    {
+        if (!clearAddOn)
+            Main.AllPlayerSpeed[playerId] = OptionSpeed.GetFloat();
+        else
+            Main.AllPlayerSpeed[playerId] = Main.RealOptionsData.GetFloat(FloatOptionNames.PlayerSpeedMod);
+    }
```

Comment:
this still isn't called anywhere 😭

Like so:
![image](https://github.com/user-attachments/assets/12caf0f2-b10b-428e-b789-3ac023f3e61a)
![image](https://github.com/user-attachments/assets/63b4a524-6161-4ca2-84dc-11aec2fab6ea)

Reply:
I gotta be real chief I don't know what you pointin' out to me 😭

Reply:
Where’s an omgcry when you need it? Ngl idk what’s he’s talking about either 💀

Reply:
In short, the function that the add-on has, which is called SetSpeed, cannot be called just like that. The code does not know when and where to play this function, since it is static, so you need to add Sloth.SetSpeed ​​where it should be played.

Reply:
I think it's enough to watch at least a quick tutorial on C# on YouTube to get an idea of ​​this

Reply:
> In short, the function that the add-on has, which is called SetSpeed, cannot be called just like that. The code does not know when and where to play this function, since it is static, so you need to add Sloth.SetSpeed ​​where it should be played.

So I added
`case CustomRoles.Sloth:
                        Sloth.SetSpeed(player.PlayerId, false);
                        break;`
underneath Flash on CustomRoleManager.cs, that sounded right, since it is essentially just Flash but speed is bad.
I do not get what I am to put in GameState.cs

Reply:
I realize that I missed attaching the picture of what I did, woops. 
![image](https://github.com/user-attachments/assets/f0377f80-af5e-463c-8a8d-9e96e7eaae5d)

Reply:
What happens in GameState.cs is the code that needs to be reproduced when removing the add-on using "Cleansed", you can just copy the Flash code and replace it with Sloth

Reply:
> What happens in GameState.cs is the code that needs to be reproduced when removing the add-on using "Cleansed", you can just copy the Flash code and replace it with Sloth

Like so? Sorry If I am wrong, I've just woken up and it's early.
![image](https://github.com/user-attachments/assets/c369282e-7e2e-452b-901f-a3f4432dea37)

Reply:
Yes

Reply:
> Yes

Okay great, then is there anything else I may need to resolve before I open this PR